docs/source/gbd2017_models/risk_attributable_causes/diabetes_mellitus/index.rst

```diff
@@ -87,6 +87,8 @@ Assumptions and Limitations
 
 4. Case definition cross-walks on FPG and HbA1c: GBD 2017 assumed that HbA1c >6.5% was equivalent to FPG >126 mg/dL.
 
+5. There are no incidence or transitions between states in this model. In the future, transition between states will be revisited using incidence of diabetes mellitus and remission.
+
 Cause Model Diagram
```

Comment:
Given the assumptions on state-to-state incidence rates (e.g. there are none), the prevalence (and subsequent mortality/morbidity) will deviate from GBD estimates - especially by the end of the simulated 5 years.  That said, our priority is to initialize with correct prevalence, since DM status impacts treatment decisions.  As such, we are willing to sacrifice some precision concerning mortality and morbidity to prioritize prevalence, which justifies the assumptions on EMR and state transition incidence.